src/foam/comics/DAOController.js

```diff
@@ -20,45 +20,82 @@ foam.CLASS({
   package: 'foam.comics',
   name: 'DAOController',
 
-  imports: [
-    'stack'
+  topics: [
+    'finished'
   ],
 
   properties: [
     {
       name: 'data',
-      hidden: true,
-      required: true
+      hidden: true
     },
     {
       name: 'predicate',
-      view: function(args, X) {
-        return {
-          class: 'foam.u2.view.RecipricalSearch',
-          of$: X.data$.dot('data').dot('of')
-        };
-      }
+      view: { class: 'foam.u2.view.RecipricalSearch' }
     },
     {
       name: 'filteredDAO',
-      view: 'foam.u2.view.ScrollTableView',
+      view: { class: 'foam.u2.view.ScrollTableView' },
       expression: function(data, predicate) {
-        return ! data ? foam.dao.NullDAO.create() :
-          predicate ? data.where(predicate) :
-          data;
+        return predicate ? data.where(predicate) : data;
       }
+    },
+    {
+      name: 'relationship'
+    },
+    {
+      name: 'selection',
+      hidden: true
+    },
+    {
+      name: 'mode',
+      value: 'browse'
     }
   ],
 
+  constants: {
+    MODE_SELECT: 'select'
```

Comment:
Document. We also have the foam.u2.ControllerMode enum, so is it confusing to have two controller modes?